docs/regeneratedocumentation.py

```diff
@@ -1,353 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Why did this file get deleted?

Reply:
I have pushed this back. Not sure why it appeared as untracked file...